Build-Wheels.py

```diff
@@ -0,0 +1,66 @@
+import json
+
+import requests
+from packaging.requirements import InvalidRequirement
+from packaging.requirements import Requirement
+
+# Global variables for branches and constraints files for the branches to be downloaded
+# (length of each list should be the same and so the positions for branch and it's specific constraints file)
+IDF_BRANCH = ['release/v5.0', 'release/v5.1', 'master']
+IDF_CONSTRAINTS = ['v5.0', 'v5.1', 'v5.2']
```

Comment:
Having two lists which dependent each other is very dangerous.

- The best would be to detect the IDF version - e.g. from the cmake file (see the implementation of `idf.py --version`). 
- Another solution would be to get it from the branch name. For master it would fall back to a variable with would define the version for master. It is still better than having two lists.